lmcache/cache_engine.py

```diff
@@ -33,6 +33,8 @@ def __init__(
         match self.config.local_device:
             case "cpu":
                 self.device = "cpu"
+            #TODO(Jiayi): Disk "uses" cuda device for now
```

Comment:
just a note, I think we should decouple the `device` and the `CacheEngine` in the future. I've added a new issue here: #48

Reply:
Totally agree.